pattern-1/bosh-release/README.md

```diff
@@ -1,87 +1,241 @@
 # BOSH release for WSO2 Identity Server deployment pattern 1
 
-This repository includes a BOSH release that can be used to deploy WSO2 Identity Server 5.4.0 deployment pattern 1
-configured to use a MySQL database on BOSH Director.
+This directory contains the BOSH release implementation for WSO2 Identity Server 5.4.0
+[deployment pattern 1](https://docs.wso2.com/display/IS541/Deployment+Patterns#DeploymentPatterns-Pattern1-HAclustereddeploymentofWSO2IdentityServer).
+
+![WSO2 Identity Server 5.4.0 deployment pattern 1](images/pattern-1.png)
+
+The following sections provide step-by-step guidelines for managing the WSO2 Identity Server 5.4.0 deployment pattern 1 BOSH release.
+
+For clarity, examples for the relevant steps have been provided for managing the BOSH release in a [BOSH Lite](https://bosh.io/docs/bosh-lite) environment.
 
 ## Prerequisites
 
-Install the following software:
+1. Install the following software:
 
-1. [BOSH CLI](https://bosh.io/docs/cli-v2.html)
-2. [Docker](https://docs.docker.com/engine/installation/)
-3. [VirtualBox](https://www.virtualbox.org/manual/ch02.html)
-4. [WSO2 Update Manager](http://wso2.com/wum)
-5. [Git client](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+    - [BOSH Command Line Interface (CLI) v2+](https://bosh.io/docs/cli-v2.html)
+    - [WSO2 Update Manager (WUM)](http://wso2.com/wum)
+    - [Git client](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+    
```

Comment:
Also, add virtualbox in the case of BOSH Lite.

Reply:
The necessary refinements have been done in https://github.com/wso2/pivotal-cf-is/pull/44.